cookbooks/fb_chrony/recipes/default.rb

```diff
@@ -22,18 +22,22 @@
 if node.centos? || node.redhat? || node.fedora?
   chrony_svc = 'chronyd'
   chrony_conf = '/etc/chrony.conf'
+  chrony_user = 'chrony'
+  chrony_group = 'chrony'
 elsif node.debian_family?
   chrony_svc = 'chrony'
   chrony_conf = '/etc/chrony/chrony.conf'
+  chrony_user = '_chrony'
+  chrony_group = '_chrony'
```

Comment:
Is this really `_chrony` (with the underscore) ?

Reply:
Yes. :(